components/divider/src/divider.js

```diff
@@ -2,23 +2,41 @@ import { colors, spacers } from '@dhis2/ui-constants'
 import PropTypes from 'prop-types'
 import React from 'react'
 
-const Divider = ({ className, dataTest, dense, margin }) => (
-    <div className={className} data-test={dataTest}>
-        <style jsx>{`
-            div {
-                display: inline-block;
-                width: 100%;
-                height: 1px;
-                background-color: ${colors.grey300};
-            }
-        `}</style>
-        <style jsx>{`
-            div {
-                margin: ${dense ? `${spacers.dp4} 0` : margin};
-            }
-        `}</style>
-    </div>
-)
+const flipMargin = (margin) => {
```

Comment:
I'm not totally sure if we want to try to remap margin provided as a prop, but this seemed probably more helpful to attempt?